Jenkinsfile

```diff
@@ -151,6 +151,16 @@ pipeline {
                 }
             }
         }
+        stage ('lxd cleanup') {
+            steps {
+                sh '''
+                set +x
+                DATE=`date -d 'now+1day' +%d/%m/%Y`
```

Comment:
Since we are looking at tomorrow's date and saying any vm before tomorrow (that means today) we will cleanup, this could trash any other PR runs that are active outside of this PR right?

Reply:
Yes, that's a great catch. I will update the PR addressing this issue